lib/features/auto-subscribe.js

```diff
@@ -26,6 +26,33 @@ function _readSubscriptionFilterFn(subscription) {
   return subscription.allowRead;
 }
 
+function _listenOnSyncEvents({rtm, filterFn}) {
+  return Promise.all([
```

Comment:
In my eyes it does not make sense to return a promise here, since registering event listeners is synchronous.

Reply:
I would make a exception in cases all functions return promises. Then even a sync function my return a promise to keep the interfaces consistent.